app/models/feed.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class Feed < ActiveRecord::Base
```

Comment:
Could we add a method to `Feed` to return the URL of the HTML version of it, like:

```ruby
def html_url
  if tumblr?
    url.sub('/rss', '')
  elsif europeana_blog?
    url.sub('/feed', '')
  else
    url
  end
end
```

That could then be used in`Home::Index#content` (view), `FeedHelper#tumblr_feed_content`, `NavigableView#navigation_global_primary_nav_blog_submenu_items` (instead of hard-coded 'http://blog.europeana.eu/'), and anywhere else we in future need such URLs.